CriticalMass/MapViewController.swift

```diff
@@ -183,18 +176,25 @@ class MapViewController: UIViewController {
 
     @objc func didReceiveInitialLocation(notification: Notification) {
         guard let location = notification.object as? Location else { return }
-        focusOnLocation(location: location)
+        focusOnCoordinate(CLLocationCoordinate2D(location))
 
-        // TODO: Replace test implemenation with controller based
-        guard Feature.events.isActive else { return }
         let coordinate = CLLocationCoordinate2D(
             latitude: location[keyPath: \Location.latitude],
             longitude: location[keyPath: \Location.longitude]
         )
-        nextRideHandler.getNextRide(around: coordinate) { result in
+        nextRideManager.getNextRide(around: coordinate) { result in
```

Comment:
We should think about anonymising the data to hide the true location.

This can be done by adding a random offset in a given range or clip decimals

Reply:
Ah. Good point. I can add that

Reply:
I am adding a random double in the Range of `(-0.009 ... 0.009)` to lat and lon